### ISSUE TITLE
control panel

### ISSUE THREAD

**Issue by yakra** [TravelMapping/Web#351]
Idea: create a new page allowing users to set
* selected user
* unit (miles / km / ft / meters)
* mapview colors
* [mapview opacity](http://forum.travelmapping.net/index.php?topic=2575)
* tile set
* etc.

and store these preferences in the cookie,
with an option to reset each to default.

**Comment by michihdeu** [comment]
please refer to #21

**Comment by jteresco** [comment]
- preference to show/not show current location

**Comment by jteresco** [comment]
I agree that putting these into browser cookies is the way to go.  If users want consistency across multiple devices and/or browsers, I expect there are tools out there to sync the cookies, etc.

Thoughts on making the control panel a popup window that can be accessed through a link in the main header bar we have on all(?) TM pages?

**Comment by michihdeu** [comment]
> Thoughts on making the control panel a popup window that can be accessed through a link in the main header bar we have on all(?) TM pages?

seconded!

**Comment by jteresco** [comment]
- default map loading location

**Comment by jteresco** [comment]
The control panel would also likely be the place to specify the subset of highway data that a user wishes to use to restrict stats/maps/rankings.
